go-controller/pkg/node/management-port.go

```diff
@@ -109,6 +115,30 @@ func (mp *managementPort) Create(nodeAnnotator kube.Annotator, waiter *startupWa
 	return cfg, nil
 }
 
+func (mp *managementPort) checkExistingMgmtPort(mgmtPortName string) error {
+	if _, stderr, err := util.RunOVSVsctl("list", "Interface", mgmtPortName); err != nil {
+		if !strings.Contains(stderr, "no row \""+mgmtPortName+"\" in table") {
```

Comment:
> --no-headings --data bare --columns type find  interface  name=

So if command fails to find interface it will return empty string, right? That is a problem, since for VF representor this command also returns empty string

Reply:
OK then you can do:
[root@ovn-control-plane ~]# ovs-vsctl --no-headings --data bare --format csv --columns type,name find  interface  name=ovn-k8s-mp0
internal,ovn-k8s-mp0